go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/tektoncd/chains
 
-go 1.18
+go 1.19
```

Comment:
Should we just bump to 1.20 while we're at it? 🤷

Reply:
My very selfish reason to stay on 1.19 is that that's currently the default version in Fedora :grin: 

Others may be on the same boat?